src/DependencyInjection/Compiler/RegisterDtoProxyCasterPass.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Fazland\DtoManagementBundle\DependencyInjection\Compiler;
+
+use Fazland\DtoManagementBundle\InterfaceResolver\ResolverInterface;
+use Fazland\DtoManagementBundle\Proxy\ProxyInterface;
+use Fazland\DtoManagementBundle\VarDumper\ProxyCaster;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\VarDumper\Caster\StubCaster;
+
+class RegisterDtoProxyCasterPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if ($container->hasDefinition('var_dumper.cloner')) {
```

Comment:
In general I prefer the `return first` style. In order to have minor complexity while reading 😄  

Can you switch to 
```php
if (! $container->hasDefinition('var_dumper.cloner')) {
    return;
}
```

Reply:
Changed